config/tool_conf.xml

```diff
@@ -31,6 +31,9 @@
     <tool file="spacy/spacy.xml"/>
     <tool file="aws/aws_comprehend.xml"/>
   </section>
+  <section id="hmgm" name="Hmgm">
+    <tool id="hmgmtest" file="hmgm/hmgm_sample.xml"/>
```

Comment:
Could you change name to "Human MGM" I'll ask Maria is she is OK with ID being hmgm. I think it's clear enough to user, but she may want to spell out. Or we can make this change later, as we work on the first HMGM tool.

Reply:
The section has been renamed to Human MGM.  I don't think the section ID is used anywhere, is it?

Reply:
Looks good. Section ID probably doesn't matter.